src/Evolution/Executables/CurvedScalarWave/EvolveWorldtubeCurvedScalarWave.cpp

```diff
@@ -9,11 +9,13 @@
 #include "Domain/Creators/TimeDependence/RegisterDerivedWithCharm.hpp"
 #include "Domain/FunctionsOfTime/RegisterDerivedWithCharm.hpp"
 #include "Evolution/Systems/CurvedScalarWave/BoundaryCorrections/RegisterDerived.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/InitialData/ZerothOrderPuncture.hpp"
 #include "Parallel/CharmMain.tpp"
 #include "Utilities/Serialization/RegisterDerivedClassesWithCharm.hpp"
 
-using metavariables = EvolutionMetavars<gr::Solutions::KerrSchild,
-                                        ScalarWave::Solutions::PlaneWave<3>>;
+using metavariables =
+    EvolutionMetavars<gr::Solutions::KerrSchild,
+                      CurvedScalarWave::AnalyticData::ZerothOrderPuncture>;
```

Comment:
Question: Does it make sense to keep the ability to use PlaneWave ID at all?

Reply:
not really. I always used a wave with amplitude 0. to get 0 initial data. But in the cases where I still need this, I can just set `Charge` to 0. Any plane wave solution is not going to be good physical solution with a scalar charge present that is not at infinity.